src/main/java/org/folio/search/configuration/KafkaConfiguration.java

```diff
@@ -43,9 +47,17 @@
     var factory = new ConcurrentKafkaListenerContainerFactory<String, ResourceEventBody>();
     factory.setBatchListener(true);
     factory.setConsumerFactory(jsonNodeConsumerFactory());
+    factory.setBatchErrorHandler(kafkaFactoryErrorHandler());
     return factory;
   }
 
+  private BatchErrorHandler kafkaFactoryErrorHandler() {
+    var errorHandler = new SeekToCurrentBatchErrorHandler();
+    // infinity retry every 20 seconds
+    errorHandler.setBackOff(new FixedBackOff(20_000, UNLIMITED_ATTEMPTS));
```

Comment:
everything looks good, except this line, maybe we will extract this property to application.yml?

Reply:
Make sense, extracted.